library/common/src/main/java/com/google/android/exoplayer2/Format.java

```diff
@@ -210,6 +211,7 @@ private Builder(Format format) {
       this.metadata = format.metadata;
       // Container specific.
       this.containerMimeType = format.containerMimeType;
+      this.prftBox = format.prftBox;
```

Comment:
seems it doesn't belong to "format", more like a metadata

Reply:
Currently I rely on `RepresentationHolders.Represetation.Format.PrftBox` to store the box.
Will consider making it as the property of `DefaultDashChunkSource`